source/dealii/amge_host.cc

```diff
@@ -27,3 +28,24 @@ template class mfmg::AMGe_host<
 template class mfmg::AMGe_host<
     3, mfmg::DealIIMatrixFreeMeshEvaluator<3>,
     dealii::LinearAlgebra::distributed::Vector<double>>;
+
+#define INSTANTIATE_COMPUTE_LOCAL_EIGENVECTORS(DIM, MESH_EVALUATOR)            \
```

Comment:
We've put all the instantiations macro in `instantiation.hpp`

Reply:
Does it make sense to move this over? Opposed to all the other macros in `instantiation.hpp` this pretty specific.